anchor/job/__main__.py

```diff
@@ -219,23 +219,48 @@ def episode_all_time_video_data(connector, web_episode_id):
 
 episodes = anchor.episodes()
 
+# We already store episode metadata from the `podcast_episode()` method above,
+# but we get additional data from the `episodes()` method (e.g. the mapping
+# between `episodeId` and `webEpisodeId`)
+logger.info(f"Sending episodesPage data to Open Podcast")
+open_podcast.post(
+    "episodesPage",
+    None,
+    episodes,
```

Comment:
I told you, you did already a good job in the past ;)

Reply:
Not sure if I understand that comment. 😅